tsconfig.json

```
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "Node",
+    "declaration": true,
+    "module": "CommonJS",
+    "sourceMap": true,
+    "strict": true,
+    "target": "es5",
```

We recently decided to change the target:
```suggestion
    "target": "ES2017",
```